src/js/components/modals/AppModalComponent.jsx

```diff
@@ -335,14 +335,14 @@ var AppModalComponent = React.createClass({
             </StoreFormGroupComponent>
             <div className="row">
               <div className="col-sm-3">
-                <FormGroupComponent
-                    attribute="cpus"
+                <StoreFormGroupComponent
+                    errorMessage={this.getErrorMessage("cpus")}
+                    fieldId="cpus"
                     label="CPUs"
-                    model={model}
-                    errors={errors}
-                    validator={appValidator}>
-                  <input min="0" step="any" type="number" required />
-                </FormGroupComponent>
+                    value={state.fields.cpus || 0.1}
```

Comment:
Why note settings the defaults directly in the app modal?
If we populate an existing app definition for editing, then we will have other "defaults".